src/resonanceReconstruction/rmatrix/legacy/unresolved/ResonanceTable.hpp

```diff
@@ -0,0 +1,78 @@
+/**
+ *  @class
+ *  @brief Unresolved resonance parameters for a specific J,pi or l,J value
+ *
+ *  Since it is required to interpolate on the resonance parameters, this
+ *  class also stores interpolation tables to provide unresolved resonances
+ *  at any energy in the unresolved resonance region.
+ *
+ *  @todo since C++17 does not allow us to decltype() something using lambdas,
+ *        a ranges::view::transformed table cannot be created and stored as
+ *        a member of the class. It could be done in the call function but then
+ *        the interpolation table would be created every time.
+ *
+ *  @todo we currently only assume linear interpolation on the parameters
+ */
+class ResonanceTable {
+
+  /* aliases */
+  template < typename XType, typename YType >
+  using Table = interpolation::Table<
+                  interpolation::table::Type<
+                    interpolation::LinearLinear,
+                    interpolation::table::search::Binary,
+			              interpolation::table::discontinuity::TakeLeft,
+                		std::vector< XType >, std::vector< YType > >,
```

Comment:
Indentation can be improved here. Looks like a mix of tabs and spaces
```suggestion
                    interpolation::LinearLinear,
                    interpolation::table::search::Binary,
                    interpolation::table::discontinuity::TakeLeft,
                    std::vector< XType >, std::vector< YType > >,
```

Reply:
I have no idea why it does that, it doesn't look like that when I open the file.

Reply:
It's because you have tabs in there. You can't see the tabs, it just looks like whitespace.